README.md

```diff
@@ -50,6 +62,20 @@ If you are not running this client on Google Compute Engine, you need a Google D
   * If you want to use a new service account, click on **Create new Client ID** and select **Service account**. After the account is created, you will be prompted to download the JSON key file that the library uses to authorize your requests.
   * If you want to generate a new key for an existing service account, click on **Generate new JSON key** and download the JSON key file.
 
+``` js
+// Authorizing on a global basis.
+var projectId = process.env.GCLOUD_PROJECT_ID; // E.g. 'grape-spaceship-123'
+
+var gcloud = require('gcloud')({
+  projectId: projectId,
+  keyFilename: '/path/to/keyfile.json'
+});
+
+// ...you're good to go! See the next section to get started using the APIs.
+```
+
+You can also set auth on a per-API-instance basis. The examples below show you
```

Comment:
Ah, good example where my self-diagnosed ADHD kicked in and I got bored of writing that sentence.